docs-2.0/7.data-security/1.authentication/1.authentication.md

```diff
@@ -16,7 +16,13 @@ Local authentication indicates that usernames and passwords are stored locally o
 
 ### Enable local authentication
 
-1. Modify the `nebula-graphd.conf` file (`/usr/local/nebula/etc/` is the default path), set `--enable_authorize=true` and save the modification.
+1. Modify the `nebula-graphd.conf` file (`/usr/local/nebula/etc/` is the default path), set the parameters as followis:
+
+  - `--enable_authorize`: Whether to enable authentication, optional values are true or false.
```

Comment:
```suggestion
  - `--enable_authorize`: Set its value to `true` to enable authentication.
```

Here we are telling the users how to enable authentication, not explaining the parameter.